avm/src/vm/opcode.rs

```diff
@@ -2078,19 +2078,19 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
   });
   io!("httpget", |args, mut hand_mem| {
     Box::pin(async move {
-      let url = HandlerMemory::fractal_to_string(hand_mem.read_fractal(args[0]));
-      let http_res = reqwest::get(&url).await;
+      let url: Uri = HandlerMemory::fractal_to_string(hand_mem.read_fractal(args[0])).try_into().unwrap();
```

Comment:
This will cause this particular opcode to potentially crash the AVM if it is fed an invalid URL, right?

Maybe we explicitly handle that and return an error result, instead?

Reply:
in the new commit, it returns an error if there's a problem parsing the URI